File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//brand服务
	app.service('brandService', function($http) {
		//this  findAll
		this.findAll = function() {
			return $http.get('../brand/findAll.do');
		}
		this.findPage = function(page, rows) {
			return $http.get('../brand/findPage.do?page=' + page + '&rows='
					+ rows);
		}
		this.add = function(entity) {
			return $http.post('../brand/add.do', entity)
		}
		this.update = function(entity) {
			return $http.post('../brand/update.do', entity)
		}
		this.findOne = function(id) {
			return $http.get('../brand/findOne.do?id=' + id);
		}
		this.del = function(selectIds) {
			return $http.get('../brand/delete.do?ids=' + selectIds);
		}
		this.search = function(page, rows, searchEntity) {
			return $http.post('../brand/search.do?page=' + page + '&rows='
					+ rows, searchEntity);
		}
		
		this.selectOptionList = function() {
			return $http.get('../brand/selectOptionList.do');
		}

	});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;
import entity.Result;

public interface BrandService {
	public List<TbBrand> findAll();

	/**
	 * 返回分页结果
	 */
	public PageResult findPage(int pageNum, int pageSize);

	/**
	 * 增加功能
	 */
	public void add(TbBrand tbBrand);

	/**
	 * 修改
	 */
	public TbBrand findOne(Long id);

	public void update(TbBrand tbBrand);

	/**
	 * 删除
	 */
	public void delete(Long[] ids);

	/**
	 * 按品牌查询
	 */
	public PageResult findPage(TbBrand tbBrand, int pageNum, int pageSize);
	//模板下拉品牌
	List<Map> selectOptionList();
}
